### PR TITLE
update fsu faculty

### PR DESCRIPTION
**Pull request by syssecfsu**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by syssecfsu** [comment]
Two changes
1. Change the affiliation of Jiawei Zhang (to UC Davis) and Xifeng Gao (to Tencent America)
2. Add Xian Fang and Xiaonan Zhang to the faculty
Thank you!

File: csrankings-j.csv [hunk 1]
```diff
@@ -1015,7 +1015,7 @@ Jiaqing Luo,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/1904.htm,NOSCHOLARP
 Jiasi Chen,Univ. of California - Riverside,http://www.cs.ucr.edu/~jiasi,NOSCHOLARPAGE
 Jiasu Sun,Peking University,http://www.sei.pku.edu.cn/people/sjs,NOSCHOLARPAGE
 Jiawei Han 0001,Univ. of Illinois at Urbana-Champaign,http://hanj.cs.illinois.edu,Kv9AbjMAAAAJ
-Jiawei Zhang 0001,Florida State University,http://jiaweizhang.net,7AkZSJsAAAAJ
+Jiawei Zhang 0001,University of California - Davis, http://jiaweizhang.net,7AkZSJsAAAAJ
```
Comment (emeryberger) [inline csrankings-j.csv]: Please correct affiliation to "Univ. of California - Davis".

File: csrankings-x.csv [hunk 2]
```diff
@@ -29,6 +29,7 @@ Xiabing Zhou,Soochow University,http://web.suda.edu.cn/zhouxiabing,NOSCHOLARPAGE
 Xiahai Zhuang,Fudan University,http://www.sdspeople.fudan.edu.cn/zhuangxiahai,XlavIMcAAAAJ
 Xiali Hei,University of Louisiana - Lafayette,http://www.xialihei.com,PzflF_cAAAAJ
 Xian Du,University of Massachusetts Amherst,https://mie.umass.edu/faculty/xian-du,8iO-e8MAAAAJ
+Xian Fang,Florida State University,https://www.cs.fsu.edu/~fan/index.html,Cy91GecAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: As far as I can tell, this is _not_ the correct DBLP entry for this person.

File: csrankings-x.csv [hunk 3]
```diff
@@ -189,6 +190,7 @@ Xiaoming Li 0001,Peking University,http://net.pku.edu.cn/~lxm,tL9zHywAAAAJ
 Xiaoming Liu 0002,Michigan State University,https://www.cse.msu.edu/~liuxm,2V2YwCMAAAAJ
 Xiaoming Zhang,Beihang University,http://scse.buaa.edu.cn/info/1038/3052.htm,NOSCHOLARPAGE
 Xiaong-Dong Li,RMIT University,https://titan.csit.rmit.edu.au/~e46507,AQewL04AAAAJ
+xiaonan Zhang,Florida State University,https://www.cs.fsu.edu/~xzhang/index.html,i2zf_1QAAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: First name needs to be capitalized.

File: csrankings-x.csv [hunk 4]
```diff
@@ -237,7 +239,7 @@ Xiaoyun Wang 0001,Tsinghua University,http://www.tsinghua.edu.cn/publish/casen/1
 Xiaozhong Liu,Worcester Polytechnic Institute,https://www.wpi.edu/people/faculty/xliu14,1BUByMcAAAAJ
 Xiapu Luo,The Hong Kong Polytechnic University,https://www4.comp.polyu.edu.hk/~csxluo,uvedrYMAAAAJ
 Xibin Zhao,Tsinghua University,http://www.thss.tsinghua.edu.cn/publish/soften/3131/2010/20101219192857877627039/20101219192857877627039_.html,NOSCHOLARPAGE
-Xifeng Gao,Florida State University,https://gaoxifeng.github.io,wSUVcN0AAAAJ
+Xifeng Gao,Tencent - America,https://gaoxifeng.github.io,wSUVcN0AAAAJ
```
Comment (emeryberger) [inline csrankings-x.csv]: Move to `old/industry.csv`.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Bump @syssecfsu

File: csrankings-x.csv (hunk 2 above)
Comment (syssecfsu) [reply csrankings-x.csv]: I am sorry, I mis-spelled her name. 

Please see her CV here https://www.cs.fsu.edu/~fan/Resume_08212020.pdf 
It lists the same publication.

**Comment by syssecfsu** [comment]
I am sorry for these issues. Please let me know if you need me to submit a new pull request. Thank you!

**Pull request merged**